### PR TITLE
Add pgAdmin to PsotgreSQL

### PR DESCRIPTION
**Pull request by acwalter**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1736)

### CODE REVIEW

**Comment by acwalter** [comment]
@MarNicGit can you take a look? The error identifies a preexisting root,PostgreSQL but it is not available in master yet. This is adding a second folder to that root. My assumption is the root is getting flagged as a dupe when this should be a subfolder, not a new root. Here is the reference https://github.com/microsoft/winget-pkgs/pull/1728

File: manifests/PostgreSQL/pgAdmin/4.22.0.0.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Homepage: https://www.pgadmin.org/
+Tags: "pgsql, sql, postgre"
+InstallerType: exe
+Switches:
+    Silent: /verysilent
+    SilentWithProgress: /verysilent
+Installers:
+    Arch: x86
+    Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
+    Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927
```
Comment (MarNicGit) [inline manifests/PostgreSQL/pgAdmin/4.22.0.0.yaml:20]:
```suggestion
Installers:
  Arch: x86
  Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
  Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927 
  Switches:
    Silent: /verysilent
    SilentWithProgress: /verysilent
```

Switches is not a root node and goes directly under the installer

File: manifests/PostgreSQL/pgAdmin/4.22.0.0.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Homepage: https://www.pgadmin.org/
+Tags: "pgsql, sql, postgre"
+InstallerType: exe
+Installers:
+  Arch: x86
+  Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
+  Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927 
+  Switches:
+    Silent: /verysilent
+    SilentWithProgress: /verysilent
```
Comment (MarNicGit) [inline manifests/PostgreSQL/pgAdmin/4.22.0.0.yaml:20]: Woops, forgot to add a dash
```suggestion
 - Arch: x86
   Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
   Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927 
   Switches:
     Silent: /verysilent
     SilentWithProgress: /verysilent

```

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**